package/yast2-control-center.spec

```diff
@@ -31,6 +31,8 @@ BuildRequires:  yast2-devtools >= 3.1.10
 BuildRequires:  pkgconfig(Qt5Core)
 BuildRequires:  pkgconfig(Qt5Gui)
 BuildRequires:  pkgconfig(Qt5Widgets)
+# For desktop file check
+BuildRequires:  yast2
```

Comment:
Hmmm, could we do it somehow differently? The problem is that this a C++ package and adding the `yast2` dependency would actually pull in the complete YaST stack (with Ruby, several Ruby gems, libyui...)

This makes the build chain longer and the package would get rebuilt much more often than really need.